cfgov/unprocessed/css/cf-enhancements.less

```diff
@@ -1127,6 +1127,218 @@ textarea {
+                    <span class="input-contains-icon_icon"></span>
+                    <span class="input-contains-icon_label"></span>
+                </button>
+            </div>
+            <input class="input__super"
+                   type="text"
+                   value="">
+        </div>
+    - name: Default icons before and after a default input, with a button.
+      markup: |
+        <div class="input-with-btn">
+            <div class="input-with-btn_input
+                        input-contains-icon">
+                <div class="input-contains-icon_before
+                            input-contains-icon_before__search">
+                    <span class="input-contains-icon_icon"></span>
+                </div>
+                <div class="input-contains-icon_after
+                            input-contains-icon_after__clear">
+                    <button class="btn btn__link btn__secondary">
+                        <span class="input-contains-icon_icon"></span>
+                        <span class="input-contains-icon_label"></span>
+                    </button>
+                </div>
+                <input type="text"
+                       value="This is some really long text to make sure that the button doesn't overlap the content in such a way that this input becomes unusable." title="Test input">
+            </div>
+            <div class="input-with-btn_btn">
+                <button class="btn">Search</button>
+            </div>
+        </div>
+    - name: Super-sized icons before and after a super input, with a button.
+      markup: |
+        <div class="input-with-btn">
+            <div class="input-with-btn_input
+                        input-contains-icon
+                        input__super">
+                <div class="input-contains-icon_before
+                            input-contains-icon_before__search">
+                    <span class="input-contains-icon_icon"></span>
+                </div>
+                <div class="input-contains-icon_after
+                            input-contains-icon_after__clear">
+                    <button class="btn btn__super btn__link btn__secondary">
+                        <span class="input-contains-icon_icon"></span>
+                        <span class="input-contains-icon_label"></span>
+                    </button>
+                </div>
+                <input class="input__super"
+                       type="text"
+                       value="This is some really long text to make sure that the button doesn't overlap the content in such a way that this input becomes unusable." title="Test input">
+            </div>
+            <div class="input-with-btn_btn">
+                <button class="btn btn__super">Search</button>
+            </div>
+        </div>
+  tags:
+    - cf-forms
+*/
+
+.input-contains-icon {
+    position: relative;
+
+    &_before {
+        .webfont-regular;
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/commit/91fbae6147b56362888b13d8c47d35d5d3284761